qt-models/divepicturemodel.cpp

```diff
@@ -98,6 +98,9 @@ QVariant DivePictureModel::data(const QModelIndex &index, int role) const
 		case Qt::DisplayPropertyRole:
 			ret = QFileInfo(entry.filename).filePath();
 			break;
+		case Qt::WhatsThisRole:
```

Comment:
When the user goes into "what's this role" this will now display the offset in seconds...?

Better use 'Qt::UserRole + 1'. If you prefer you can also give it a symbolic name in the class definition like:
```
        enum {
                OffsetRole = Qt::UserRole + 1,
                LengthRole = Qt::UserRole + 2
        };
```